dev/cnf/oss_dependencies.maven

```diff
@@ -73,13 +73,15 @@ commons-cli:commons-cli:1.4
 commons-codec:commons-codec:1.10
 commons-codec:commons-codec:1.11
 commons-codec:commons-codec:1.14
+commons-codec:commons-codec:1.15
 commons-codec:commons-codec:1.3
 commons-codec:commons-codec:1.6
 commons-collections:commons-collections:3.2.2
 commons-io:commons-io:2.8.0
 commons-lang:commons-lang:2.4
 commons-logging:commons-logging:1.1.1
 commons-logging:commons-logging:1.1.3
+commons-logging:commons-logging:1.2
```

Comment:
Given that the goal of the PR is to use common versions, can the other versions be removed from this file?

Reply:
going to do removal in a third PR, after I have switched CL to use the new bundles